pyproject.toml

```
@@ -3,14 +3,27 @@ name = "mozmlops"
 version = "0.1.0"
 description = "A package for getting your models into production"
 authors = ["Mozilla MLOps"]
-license = "Proprietary"
+license = { file = "LICENSE" }
```

```suggestion
license = "MPL-2.0"
```
Sorry, I was actually wrong on this 😮‍💨 Poetry doesn't support this syntax apparently.

---

DINGDANGIT I was so excited about this too 😂 conveniently we're not ever likely to need to change this line anyway.

Similarly for the pytest options I had to do some futzing around to get poetry to accept the syntax. Nothing huge.